pkg/scene/builder.go

```diff
@@ -85,6 +88,13 @@ func (b *Builder) WidgetSystem(widgetSystem *WidgetSystem) *Builder {
 	return b
 }
 
+// WidgetAlignSystem _
```

Comment:
Sorry just to be clear, you want ALL the `// xxx _` comments to be removed from this file?

Reply:
Yes, but it is hard to edit many files at once because many of them have such comments. Therefore, it would be nice if you could delete the comments left in the file when you edit it.

Incidentally, the reason why such comments are still there is that before the introduction of golangci-lint, all types and functions had to be commented or else a warning would appear.

Reply:
Noted! From now on when I see comments like that I will remove them.